### PR TITLE
added show memory

### LINKED ISSUE #37: Create Stub function for show memory
Work required for show memory #15

### LINKED ISSUE #58: Create stub functions for show memory detail
Work required for show memory #16

### PR DESCRIPTION
**Pull request by scotter567**
finishing up stub function

### CODE REVIEW

**Comment by claesmk** [comment]
Merge conflicts need to be resolved before I can merge this

File: asa_parser/src/asa_parse.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def show_cpu_usage(self):
         return json.dumps({'text': self.get_show_section('cpu usage')})
 
     def show_memory_region(self):
-        """Parser for show memory region"""
+        """Parser for show_memory.txt region"""
```
Comment (claesmk) [inline asa_parser/src/asa_parse.py]: Did you mean to put a .txt here? Was that a cut/paste problem?

**Review by claesmk — CHANGES_REQUESTED** [review]

File: asa_parser/src/asa_parse.py (hunk 1 above)
Comment (scotter567) [reply asa_parser/src/asa_parse.py]: I believe this was a cut and paste problem. I'll remove the .txt should I re commit?

Comment (claesmk) [reply asa_parser/src/asa_parse.py]: yes - do a commit and a push to your fork and the pull request should update

**Review by claesmk — APPROVED** [review]

**Pull request merged**